contracts/v3/Onboarding.sol

```diff
@@ -54,15 +55,14 @@ contract OnboardingContract {
         proposal.amount = amount;
         proposal.sharesRequested = sharesRequested;
         proposal.applicant = newMember;
-        
     }
 
     function sponsorProposal(uint256 proposalId) external {
         IProposalContract proposalContract = IProposalContract(dao.getAddress(PROPOSAL_MODULE));
         proposalContract.sponsorProposal(dao, proposalId, msg.sender);
     }
 
-    function processProposal(uint256 proposalId) external {
+    function processProposal(uint256 proposalId) external nonReentrant {
```

Comment:
does processproposal call an external contract? If not we don't need this nonReentrant

Reply:
done